src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/HealthDTO.java

```diff
@@ -0,0 +1,22 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto;
+
+public class HealthDTO {
```

Comment:
Can we please go back to where we stated what kind of DTO it is?
Like HealthGetDTO or HealthPutDTO?

I know, I said anytime anywhere that I like it but this was not a well thought through idea from my side, sorry for that.